cantabular-import/README.md

```diff
@@ -7,16 +7,13 @@ Expects you to have environment variables `zebedee_root` and
 
 Expects your services to be in the expected relative path
 
-dp-dataset-api expects a connection to Neptune via ssh. Won't break
-import but health checks will fail and produce annoying messages.
-
 # Bring Up Cantabular Import Services #
 
-`sudo docker-compose up`
+`sudo -E docker-compose up` or `./run.sh`
```

Comment:
Really? It's just the command to start the services.. in fact run.sh is just that command.. just thought it might be good to know what the script does without looking in it

Reply:
Yes, really... don't assume anything about your audience (within reason)